### ISSUE TITLE
10.7 Beta Errors

### ISSUE THREAD

**Issue by vahtos** [exeGesIS-SDM/GISquirrel#28]
For some reason the installation of GISquirrel v1_3_4698 is setting the installation directory (C:\Program Files (x86)\ESDM\GISquirrel) as read-only, with only read access for the user (non-administator). It is doing the same thing to the file: C:\ProgramData\GISquirrel\GISquirrel.config

This has occurred on two different machines that I have done the install on. It is causing errors such as the one below, and making the extension unusable in Arcmap.

![image](https://user-images.githubusercontent.com/30844190/71032588-a2d8be80-20e3-11ea-92fa-88a94554bbda.png)

**Comment by CrispinF** [comment]
Thanks @vahtos I observed that same error on one of our test machines too (@HartmutT this was "VM-NRHE"), but actually I don't think it's caused by folder permissions.
However I did find I could use almost all functionality, including importing layers.  I think it was the Feature Class Administrator that wouldn't load and threw this error. We will look into it.

**Comment by vahtos** [comment]
The error in the screenshot was resolved by changing the file permissions on C:\ProgramData\GISquirrel\GISquirrel.config


The error for opening the Feature Class Administrator was solved by changing the directory and file permissions for all folders in the default install directory.

This behavior and these workarounds held true for both machines I tested it on.

Glad you're looking into this. Thank you. Let me know if there is any more information I can provide.
